standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -3658,6 +3658,20 @@ public boolean deletePartitionColumnStatistics(String catName, String dbName, St
         tableName, partName, colName, engine);
   }
 
+  @Override
+  public boolean deletePartitionMultiColumnStatistics(String dbName, String tableName, String partName,
```

Comment:
I don't like introducing two APIs for the same purpose. Can we remove this API and use the other one?

Reply:
I agree with this. We should remove one of them, retain the one that takes the Request Object, rename it to something more generic like DeleteColumnStatisticsRequest, make the PartitionName optional and use this value to determine whether to delete the Table column stats or Partition columns stats.